hbase_sql/src/main/java/com/splicemachine/olap/OlapServerSubmitter.java

```diff
@@ -598,29 +607,45 @@ static String quoteForBatchScript(String arg) {
     private String prepareCommands(String exec, String parameters, String sparkYarnQueue) throws IOException {
         StringBuilder result = new StringBuilder();
         result.append(exec);
-        for (Object sysPropertyKey : System.getProperties().keySet()) {
```

Comment:
This changes how we handle Spark configuration significantly, we need an upgrade path without breaking all existing installations.

Reply:
This was based on our discussion - Olap server parameters are set as HMaster Java options. This is sometimes confusing for customer. We decided to move everything to hbase-site.xml